src/ui/wxWidgets/AddEditPropSheetDlg.h

```diff
@@ -288,8 +288,14 @@ class AddEditPropSheetDlg : public wxPropertySheetDialog
 
   void ItemFieldsToPropSheet();
   void SetupDCAComboBoxes(wxComboBox *pcbox, short &iDCA, bool isShift);
-  void UpdateExpTimes();        // entry -> controls
+  void InitializeExpTimes();        // entry -> controls
   void SetXTime(wxObject *src); // sync controls + controls -> entry
+
+  // Today returns time part == 0
+  wxDateTime TodayPlusInterval() { return wxDateTime::Today().Add(wxDateSpan(0, 0, 0, m_ExpirationTimeInterval)); };
```

Comment:
Stylistic preference, but for simple functions like this it's better IMO to not have side effects and instead take all inputs as parameters. It's just not obvious from the prototype what interval we're adding. Also, the function would be more reusable with a parameter for interval.

Reply:
Good point.  Changed.